sympy/functions/combinatorial/factorials.py

```diff
@@ -728,4 +726,5 @@ def _eval_rewrite_as_gamma(self, n, k):
         return gamma(n + 1)/(gamma(k + 1)*gamma(n - k + 1))
 
     def _eval_is_integer(self):
-        return self.args[0].is_integer and self.args[1].is_integer
+        if self.args[0].is_integer is True:
```

Comment:
Would ``self.args[0].is_integer`` alone be sufficient here, or are there examples where the results differ?